### PR TITLE
Update CI credentials docs to point to new GCP project

### PR DESCRIPTION
**Pull request by girarda**
## What
* Update the docs to point to `ab-connector-integration-test` since `dataline-integration-testing` is being deprecated according to https://www.notion.so/Cloud-Project-Structure-9a4a73af6f1d48b8b87f6380da10e722#03ee0a30b798403cb1faa1e77f7643a4

### CODE REVIEW

**Review by wennergr — APPROVED** [review]
LGTM

**Comment by airbyte-oss-build-runner** [comment]
##  destination-s3 test report (commit [`89d3d85cd1`](https://github.com/airbytehq/airbyte/pull/30813/commits/89d3d85cd1b445cfd57dc5fb189ea95e226a7cdc)) - ❌

⏲️  Total pipeline duration: 02mn06s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Build connector tar                                                   | ✅        |
| Build destination-s3 docker image for platform linux/x86_64           | ❌        |
| Validate airbyte-integrations/connectors/destination-s3/metadata.yaml | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ❌        |
| QA checks                                                             | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/alex_update_ci_credentials_docs/1695855725/89d3d85cd1b445cfd57dc5fb189ea95e226a7cdc/destination-s3/0.5.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:89d3d85cd1b445cfd57dc5fb189ea95e226a7cdc)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-s3 test
```

File: airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/S3DestinationStrictEncrypt.java [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.google.common.annotations.VisibleForTesting;
-import io.airbyte.cdk.integrations.destination.s3.S3DestinationConfigFactory;
-import io.airbyte.cdk.integrations.destination.s3.S3DestinationConfig;
 import io.airbyte.cdk.integrations.destination.s3.S3BaseChecks;
+import io.airbyte.cdk.integrations.destination.s3.S3DestinationConfig;
```
Comment (bnchrch) [inline airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/S3DestinationStrictEncrypt.java:10]: Looks like this was made during a dirty master state.

Lets revert the autoformat commit and merge latest master

**Review by bnchrch — APPROVED** [review]
lgtm!

**Pull request merged**